build/toolchain/archs/x86_64.mk

```diff
@@ -9,8 +9,7 @@ ARCH_KCFLAGS = \
 	-mno-red-zone \
 	-mcmodel=kernel
 
-ARCH_CFLAGS = \
```

Comment:
why are you removing sanitizer :^(

Reply:
Oops my bad, I removed them because it slowed down rendering a lot

Reply:
I'll rollback this change